libs/blocks/video/video.css

```diff
@@ -5,4 +5,132 @@ a[href*='.mp4'].hide-video {
 video {
   max-width: 100%;
   height: auto;
+  object-fit: cover;
+}
+
+:is(.marquee, .aside.split) .pause-play-wrapper img.accessibility-control {
+  min-height: 40px;
+}
+
+.video-container {
+  display: flex;
+  position: relative;
+  height: 100%;
+  width: fit-content;
+  margin: auto;
+}
+
+:is(.aside, .marquee, .quiz-marquee) .video-container {
+  width: auto;
+}
+
+.brick-media .video-container {
+  width: 100%;
+  height: 100%;
+  border-radius: inherit;
+}
+
+.pause-play-wrapper {
+  display: flex;
+  width: fit-content;
+  margin: auto;
+}
+
+.video-container .pause-play-wrapper {
+  width: 45px !important;
+  height: 45px !important;
+  position: absolute;
+  bottom: 2%;
+  right: 2%;
+  margin: 0;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  z-index: 2;
+  cursor: pointer;
+}
+
+.video-container .pause-play-wrapper .offset-filler {  
+  display: inherit;
+  justify-content: center;
+  align-items: center;
+  width: 40px;
+  height: 40px;
+  border-radius:inherit;
+  background: #242424;
+}
+
+:is(.marquee:not(.light), .dark) .video-container .pause-play-wrapper .offset-filler {
+  border: 2px solid white;
+}
+
+:is(.marquee:not(.light), .dark) .video-container .pause-play-wrapper:focus .offset-filler {
+  outline: 1px solid #000;
+}
+
+.video-container .pause-play-wrapper .offset-filler:hover {
+  background: #000;
+  outline: var(--color-accent-focus-ring) solid 2px;
+}
+
+.video-container .pause-play-wrapper:focus .offset-filler {
+  border: 2px solid white;
+  outline: 1px solid white;
+}
+
+.video-container .pause-play-wrapper:focus-visible {
+  background: #000;
+  outline: var(--color-accent-focus-ring) solid 2px;
+}
+
+:is(.editorial-card, .hero-marquee, .marquee):not(:has(.video-container)) .pause-play-wrapper,
+.editorial-card .video-container {
+  width: auto;
+  height: 100%;
+}
+
+.brick .brick-media:not(:has(.video-container)) .pause-play-wrapper {
+  border-radius: 0;
+  border-top-right-radius: inherit;
+  border-bottom-right-radius: inherit;
```

Comment:
Could the video be on the other side? How does this behave on RTL? During my quick check, it would make it lose its border radius values altogether.

Reply:
yes, the video can be on the other side.
yes, it would lose the border-radius but would maintain the structure of the brick block.